charts/stellar-disbursement-platform/minimal-values.yaml

```diff
@@ -61,6 +58,7 @@ sdp:
       ANCHOR_PLATFORM_OUTGOING_JWT_SECRET: #required for mySdpToAnchorPlatformSecret
       DATABASE_URL: #required
       DISTRIBUTION_SEED: #required
+      DISTRIBUTION_ACCOUNT_ENCRYPTION_PASSPHRASE: #required
```

Comment:
This is required because we're using `DISTRIBUTION_SIGNER_TYPE=DISTRIBUTION_ACCOUNT_DB`